File: c_files/calc1.c
#include <stdio.h>
void main()
{
	float a, b, c, op;
	
	printf("Enter a Number: ");
	scanf("%f", &a);

	printf("Enter a Number: ");
	scanf("%f", &b);

	printf("Enter 1 for addition, 2 for subtraction, 3 for multiplication & 4 for division.\n");
	printf("Enter Operator: ");
	scanf("%f", &op);


	if (op == 1)
	{
		c = a + b;
		printf("Answer = %f", c);
	}
	else if (op == 2)
	{
		c = a - b;
		printf("Answer = %f", c);
	}
	else if (op == 3)
	{
		c = a * b;
		printf("Answer = %f", c);
	}
	else if (op == 4)
	{
		c = a / b;
		printf("Answer = %f", c);
	}
	else
	{
		printf("Error!!");
	}
}

File: python/learning/calc1.py
while True:	 # infinite while loop
	a = float(input("Enter 1st. Number: "))  # Takes input from user and stores the value in the variable 'a'
	b = float(input("Enter 2nd. Number: "))  # Takes input from user and stores the value in the variable 'b'

	if a == 0.0 and b == 0.0:  # If condition is true statement will be executed
		break
		
	print("\nOperations: \n '+' --> addition\n '-' --> subtraction\n '*' --> multiplication\n '/' --> division\n\n")  # Prints out the text in "  "
	op = input("Enter Operator: ")  # Takes input from user and stores the value in the variable 'op'

	if op == '0':  # If condition is true statement will be executed
		break
	
	if op == '+':  # If condition is true statement will be executed
		c = a + b
	elif op == '-':  # If condition is true statement will be executed
		c = a - b
	elif op == '*':  # If condition is true statement will be executed
		c = a * b
	elif op == '/':  # If condition is true statement will be executed
		c = a / b
					
	print("Answer = " + str(c) + "\n")  # Prints out the text in "  "
